uber/templates/base.html

```diff
@@ -21,6 +21,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     {% block sectionStyle %}{% endblock %}
     {% block additional_styles %}{% endblock %}
+    {% if 'preregistration' not in c.PAGE_PATH %}
```

Comment:
Unfortunately there's no way to do this inside the includes themselves, which is what I would've preferred.